### PR TITLE
Csm test fixes

### PR DESCRIPTION
**Pull request by aditipande96**
Signed-off-by: Aditi Pande <aditi.pande@seagate.com>

# Problem Statement
- Problem statement

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
-  [ ] New/Affected tests are executed on Latest Build
-  [ ] Attach test execution logs
-  [ ] Collection tested and no collection error introduced (`pytest --local True --collect-only`)

# Review Checklist 
  Checklist for Author
- [ ] JIRA number/GitHub Issue added to PR
- [ ] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
-  [ ] If change in any common function, make sure to update all calls and execute all affected tests.

# Documentation
  Checklist for Author
-  [ ] Changes done to ReadMe / WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

**Comment by aditipande96** [comment]
Test Logs:
PASSED tests/csm/rest/test_csm_users.py::TestCsmUser::test_6220
PASSED tests/csm/rest/test_csm_users.py::TestCsmUser::test_25278
PASSED tests/csm/rest/test_csm_users.py::TestCsmUser::test_25280
PASSED tests/csm/rest/test_csm_users.py::TestCsmUser::test_25282
PASSED tests/csm/rest/test_csm_users.py::TestCsmUser::test_5018
Skipped tests: test_5014, test_5023, test_7360, test_1228, test_1235, test_1227

Collection logs:
========= 2154 tests collected in 10.85s ========

File: config/csm/test_rest_csm_user.yaml [hunk 1]
```diff
@@ -178,16 +178,16 @@ test_7362:
   password: 'qiLA1WIqNyX0Uy4gz1RuqEiM2ZCVHAok0Ht6y0HS04E='
 
 test_25278:
-    error_code: 4101
+    error_code: 4104
     message: "{} can not update the role for {}"
```
Comment (kachhwahadivya) [inline config/csm/test_rest_csm_user.yaml:182]: message should be read from response yaml file

File: config/csm/test_rest_csm_user.yaml [hunk 2]
```diff
@@ -178,16 +178,16 @@ test_7362:
...
     message_id: "update_not_allowed"
 test_25280:
-    error_code: 4101
+    error_code: 4104
     message: "{} can not update the password for {}"
```
Comment (kachhwahadivya) [inline config/csm/test_rest_csm_user.yaml:186]: message should be read from response yaml file

File: config/csm/test_rest_csm_user.yaml [hunk 3]
```diff
@@ -178,16 +178,16 @@ test_7362:
...
     message_id: "update_not_allowed"
     current_password: "puUqs4Ns/l/fh6teILWN1FPFu3RZogVZ0SbtsMpbkOA="
 test_25282:
-    error_code: 4101
+    error_code: 4104
     message: "{} can not update the email id for {}"
```
Comment (kachhwahadivya) [inline config/csm/test_rest_csm_user.yaml:191]: message should be read from response yaml file

File: tests/csm/rest/test_csm_users.py [hunk 4]
```diff
@@ -1981,66 +1984,6 @@ def test_6220(self):
         self.log.info(
             "Step 2: Verified that csm manage user is not able to create duplicate csm user")
```
Comment (kachhwahadivya) [inline tests/csm/rest/test_csm_users.py:1985]: U need to remove this line, Can u create new test with below steps and write in backlog sheet.

Comment (aditipande96) [reply tests/csm/rest/test_csm_users.py:1985]: will create a new test. Why does this line need to be removed?

File: config/csm/test_rest_csm_user.yaml (hunk 3 above)
Comment (kachhwahadivya) [reply config/csm/test_rest_csm_user.yaml:191]: Error code changes are also taken care in above PR. So we can skip from here

**Review by kachhwahadivya — APPROVED** [review]

**Review by RajeshPratapSInghChouhan — APPROVED** [review]

**Review by seagate-sarang-sawant — APPROVED** [review]

**Pull request merged**